### PR TITLE
add Keywords for Salt

### PR DESCRIPTION
**Pull request by goncalvesmachadoc**
First steps to add new keywords for salt precipitation module.

### CODE REVIEW

File: src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TEMPVD [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{"name" : "TEMPVD", "sections" : ["PROPS"],
-"items" : [
-  {"name" : "DEPTH", "value_type" : "DOUBLE", "dimension" : "Length"},
-  {"name" : "TEMP", "value_type" : "DOUBLE", "dimension" : "Temperature"}]}
-
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TEMPVD]: Hmm - the changes to the `TEMPVD`keyword are they intentional?

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Apart from the changes to the `TEMPVD` keyword which I was a little surprised by this looks perfectly OK.

File: src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TEMPVD (hunk 1 above)
Comment (goncalvesmachadoc) [reply src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TEMPVD]: I will fix this.

**Pull request merged**